templates/templates/base.html

```diff
@@ -53,7 +67,7 @@
   <!-- footer content goes here -->
   {% include "templates/footer.html" %}
 
-<script src="/static/dist/global-nav.js"></script>
+<script src="/static/dist/global-nav.js" async defer></script>
```

Comment:
```suggestion
<script src="/static/dist/global-nav.js" defer></script>
```

You should use `async` or `defer` - in this case, use `defer`